### DISCUSSION TITLE
Supabase function vs Deno Deploy feature downgrade

### DISCUSSION THREAD

**Discussion by abhay187** [supabase/supabase#11448] [Questions]
It is well known that the Supabase Edge function uses Deno deploy underneath event both have the same pricing of $2/million. Still, there is a major downgrade in what Supabase is offering us vs Deno deploy. I have added screenshots of both. I felt cheated as a customer. Would like to know other people and especially people at the supabase why there is a major downgrade in terms of features and script size.

<img width="879" alt="Screenshot 2023-01-04 at 11 23 19 PM 1" src="https://user-images.githubusercontent.com/61496865/210620891-12882b54-5c30-4b47-8c5a-6c5af6e3820f.png">
<img width="1121" alt="Screenshot 2023-01-04 at 11 36 14 PM" src="https://user-images.githubusercontent.com/61496865/210620906-f79602b0-a73d-4b37-8620-903fd60dbf8d.png">

**Comment by abhay187** [comment]
What supabase team thinks about this move? Also, why not mention all the features on the pricing page itself?
@kiwicopple @awalias @inian @alaister @RamiroND @MildTomato @joshenlim @dshukertjr @thorwebdev

**Comment by laktek** [comment]
@abhay187 Yes, Supabase Edge Functions are currently sub hosted with Deno Deploy. So in terms of the feature list there's no real downgrade. We will update our pricing page to provide a better break down of features in the coming days.

Here's a quick breakdown of how the Supabase Edge Function offering compares to Deno Deploy:

* Supabase don't have any restrictions on no of deployments per hour.
* Also, Supabase don't restrict the no of logs and crash reports. The free plan has 1 day of log retention, and if you upgrade to Pro plan it extends to 7 days.
* For CPU time, Memory, ES modules per deployment and Environment variables, we do match up to what Deno is offering. 
* For the script size, we are currently limited by our function blob storage - we are doing some infrastructure updates planned to lift that limit.
